book.adoc

```diff
@@ -0,0 +1,276 @@
+= ownCloud Administration Manual
+The ownCloud Developers <docs@owncloud.com>
+10.0.10, {docdate}
+:appversion: Release 10.0.10
+:source-highlighter: rouge
+:homepage: https://github.com/owncloud/docs
+:listing-caption: Listing
+:toc: 
+:toclevels: 1
+:icons: font
+:icon-set: octicon
+
+include::modules/ROOT/pages/index.adoc[leveloffset=+1]
```

Comment:
``ROOT`` how ugly... :neckbeard:
If we do not fix that one first, it will be imho a big pain afterwards...

Reply:
I disagree. At first I was quite put off by it, but after a while of getting used to it, it makes sense to me.